### ISSUE TITLE
New UI Issues: AVL

### ISSUE THREAD

**Issue by kechinn** [WSDOT/HATS#909]
Latest Vehicle Records:
- This may be something I missed that Joe wanted or you haven't gotten to yet...Export Latest Vehicle Records now only available as an export to excel and not a web page with user friendly column names

Raw Data Report:
-  ~~AVL Vehicle Number missing * symbol (logic still exists so it wont let you search without)~~
- ~~Raw Data Report: not exporting any data to excel~~

Add/Edit Maintenance Vehicle/Roadside Equipment:
- ~~When editing, you are required to set a Region in order to set Activity Flag as False (not a big deal, but seems kinda weird. Maybe it allow you to enter that stuff during Add process rather than Edit process).~~

**Comment by jhudsonWA** [comment]
@joeschmit:

Re: "This may be something I missed that Joe wanted or you haven't gotten to yet...Export Latest Vehicle Records now only available as an export to excel and not a web page with user friendly column names"

I changed this. It had friendly name columns in the current version but was way too wide for bootstrap. Bootstrap wants a UI that can fit within 100% of the browser window to allow for the side to side nav bar. If we include a readable table on this screen it requires a  width of 3000+ pixels. This was fine with the old nav bar but terrible for the new as it cuts off at the edge of the browser. I settled on just an export to make it work. The old table showed every column in the database, which the export now does.

Thoughts?

**Comment by jhudsonWA** [comment]
Raw data, you are in QA so no data will export as we are not running the services there. It will be fine in production.

**Comment by jhudsonWA** [comment]
Add/Edit Maintenance Vehicle/Roadside Equipment: 

Region is set from the truck name on add, so no need to set. I've added the * on edit. Most vehicles already have a region. I think you tried to edit one of the trucks from the initial load where region could not be figured out.

**Comment by jhudsonWA** [comment]
@kechinn see notes, ready for retest.

**Comment by jhudsonWA** [comment]
@joeschmit: repinging on this: 

http://hatsqa.wsdot.loc/Maintenance/Management/Activities/AVL/LatestVehicleRecords

Re: "This may be something I missed that Joe wanted or you haven't gotten to yet...Export Latest Vehicle Records now only available as an export to excel and not a web page with user friendly column names"

I changed this. It had friendly name columns in the current version but was way too wide for bootstrap. Bootstrap wants a UI that can fit within 100% of the browser window to allow for the side to side nav bar. If we include a readable table on this screen it requires a width of 3000+ pixels. This was fine with the old nav bar but terrible for the new as it cuts off at the edge of the browser. I settled on just an export to make it work. The old table showed every column in the database, which the export now does.

Thoughts?
